src/redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import MissionService from '../../services/missionsService';
+
+// Actions
+const FETCHED_MISSONS = 'space-travelers/missions/FETCHED_MISSONS';
+
+// Actions Creator
+export const getMissions = createAsyncThunk(FETCHED_MISSONS, MissionService.fetchMissions);
+
+// Slice
+const missionSlice = createSlice({
+  name: 'missions',
+  initialState: {
+    missions: [],
+  },
+  extraReducers: (builder) => {
+    builder.addCase(getMissions.fulfilled, (state, action) => ({
+      ...state,
+      missions: action.payload,
+    }));
+  },
+});
```

Comment:
[optional]:
Maybe you can try to handle other cases in the future, for example, if someone has a low-speed internet, you can inform him/her, that the page is loading by handling the action: `getMissions.pending` and then showing a progress bar when the state is in loading shape.
The same thing if something wrong happened and the request returned an error. you can maybe handle it `getMissions.rejected`